bin/spark-submit.sh

```diff
@@ -0,0 +1,6 @@
+/var/lib/spark/bin/spark-submit \
```

Comment:
maybe add those as part of the readme. I also wonder whether this cli should just be called something else, `bin/jsonpedia` and the call to spark would be handled under the hood.